resources/views/components/molecules/_m-listing----stories-listing.blade.php

```diff
@@ -27,7 +27,13 @@
                 @else
                     <span class="default-img"></span>
                 @endif
-                <span class="m-listing__img__overlay"></span>
+                <span class="m-listing__img__overlay" style="display: block">
```

Comment:
Instead of making a `<span>` display as a block, could this be a `<div>` instead?

Reply:
trueee